akka-typed/src/main/scala/akka/typed/ActorContext.scala

```diff
@@ -105,8 +105,8 @@ class StubbedActorContext[T](
       case Some(inbox) ⇒ inbox.ref == child
     }
   }
-  override def watch[U](other: ActorRef[U]): ActorRef[U] = other
-  override def unwatch[U](other: ActorRef[U]): ActorRef[U] = other
+  override def watch(other: ActorRef[_]): Unit = ()
```

Comment:
Hmmm... is this class touched by Java too? If so then the `[U]` makes it more usable.
Return type change is good.